.travis.yml

```diff
@@ -28,4 +28,6 @@ after_failure:
     - cd ../..
     - pwd
     - ls -la $HOME
-    - ls -la $HOME/android-sdk
\ No newline at end of file
+    - ls -la $HOME/android-sdk
+
+dist: trusty
```

Comment:
nit, would be easier to read the environment if `dist: trusty` was one of the first few lines